### DISCUSSION TITLE
https://api.powerbi.com/v1.0/myorg 404 Not Found

### DISCUSSION THREAD

**Discussion by JackCoxer** [migueesc123/PowerBIRESTAPI#144] [Q&A]
I have added the connector in to Power BI Desktop, and i'm able to add tables and view the data in them within Power Query, however when I try and add the tables I get an error message:-

Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: Exception of type 'Microsoft.Mashup.Engine.Interface.ResourceAccessForbiddenException' was thrown.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.

I tried adding in the GETData function, as I thought that may be the issue, however the error message i'm getting is:-

DataSource.Error: Web.Contents failed to get contents from 'https://api.powerbi.com/v1.0/myorg' (404): Not Found
Details:
    DataSourceKind=PowerBIRESTAPI
    DataSourcePath=PowerBIRESTAPI
    Url=https://api.powerbi.com/v1.0/myorg

Can anyone shed any light? I'm new to APIs etc so any help would be greatly appreciated.

**Comment by migueesc123** [comment]
hey! can you share any repro steps?

**Reply by JackCoxer** [reply]
Good morning,

Think it was user error on my part, I was trying to add too much data. I've managed to get it working for mostly everything I need.

While i've got you, do you know if it's possible to use this to get usage data from our environment? I'd like to be able to set up a usage page similar to what you're able to pull from service within the admin view i'm creating.

**Reply by migueesc123** [reply]
I haven't read the documentation in a long time, but if you can find an endpoint that provides the data that you're looking for, it should be pretty straightforward to either use the GETData function inside the connector or to add it to the connector itself
